common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -601,6 +600,13 @@ def asset(location, subs_id, lang='en', filename=None):
 
         `location` is module location.
         """
+        # HACK Warning! this is temporary and will be removed once edx-val take over the
+        # transcript module and contentstore will only function as fallback until all the
+        # data is migrated to edx-val. It will be saving a contentstore hit for a hardcoded
+        # dummy-non-existent-transcript name.
+        if NON_EXISTENT_TRANSCRIPT in [subs_id, filename]:
+            raise NotFoundError
```

Comment:
I think we should check it on final/last level which is get_asset.  can we move these checks there ?

Reply:
I don't see any usage of `Transcript.get_asset` that may slip this through.